### DISCUSSION TITLE
Alerting: Simplify the default template

### DISCUSSION THREAD

**Discussion by phemmer** [grafana/grafana#75824] [General]
Firstly, this is supposed to be in the "Alerting" category, but github won't let me create anything other than a "poll" discussion.

----

So we were using version 8.5, and the alert messages (specifically to Slack), were very simple, concise, and easy to read. After upgrading to 9.4, they've become a lot more verbose, and yet somehow have lost critical information that made the alerts useful.

Lets look at an example:
![Screenshot_2023-10-02_09-08-54](https://github.com/grafana/grafana/assets/1826947/bbffdc73-b541-42b2-890f-f511abc25ca9)
^old
![Screenshot_2023-10-02_09-10-22](https://github.com/grafana/grafana/assets/1826947/7b391f85-c0b8-4488-bda1-391caa5a9641)
^new

The only complaint I had with the old format was it didn't show the unit of the metric. But that's minor.
The new format however is extremely verbose, dumping way too much unnecessary information. But the worst issue is that I no longer can tell which host generated the alert. The old message clearly indicated the name of the series which the value is from. The new alert just says "B0".

I'm sure I can probably create a custom template to fix this. But now I have to get a Ph. D. in Grafana Alerting to figure out how to create a new template which mimics the old one.
I'm all for adding more customization capabilities, but just because you can cram every last iota of information into the default template doesn't mean you should. IMHO it would be far more sensible for the default template to contain the minimum pertinent information, and then for the other information which is needed <1% of the time, provide a link which takes you to a browser view with details.

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move https://github.com/grafana/grafana/discussions/73424.

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!
